src/utils/helpers.ts

```diff
@@ -13,3 +15,7 @@ export function isComponentFunctionType(
 ): element is Function {
   return typeof element === 'function' && React.isValidElement(element());
 }
+
+export const getLastDataItemProperty = (data: BreadcrumbItemData[], propertyPath: string) => {
```

Comment:
a small problem here is that this function is listed under the global helpers. Thus anyone that will try to use it will have to "feed it" that exact type of data. I feel that this is more of a local helper for breadcrumb.

Also this can be avoided by using lodash pick or omit 
```_.pick(_.last(data), ['label', 'onChangeHandler', 'options'])```

Reply:
I agree this is more like a local helper. Should i create a util.ts file inside the breadcrumb directory ? 

Great idea on refactoring it this way.